qlib/data/ops.py

```diff
@@ -118,6 +151,8 @@ def __init__(self, feature, func):
 
     def _load_internal(self, instrument, start_index, end_index, *args):
         series = self.feature.load(instrument, start_index, end_index, *args)
+        if series.dtype != np.float32:
```

Comment:
This checker may be required in PairOperator,  Rolling, and PairRolling.
The checking can happen during the stage of initialization if operations like 'cat' are kept.